sql/src/test/java/io/crate/analyze/SelectStatementAnalyzerTest.java

```diff
@@ -1885,4 +1885,30 @@ public void testGroupByGeoShape() throws Exception {
         expectedException.expectMessage("Cannot GROUP BY 'shape': invalid data type 'geo_shape'");
         analyze("select count(*) from users group by shape");
     }
+
```

Comment:
maybe its time to create dedicated test classes for select statement variants ;)